x-pack/plugins/security_solution/public/detection_engine/rule_management/components/rule_details/three_way_diff/final_readonly/fields/name/name.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiDescriptionList } from '@elastic/eui';
+import * as ruleDetailsI18n from '../../../../translations';
+import type { RuleName } from '../../../../../../../../../common/api/detection_engine';
+
+interface NameProps {
```

Comment:
nit: Move internal entities down in the file.

Reply:
Thanks! Added in https://github.com/elastic/kibana/pull/192342/commits/f89e47613bf528de50571615fbe2a0e6b8c53d7a